varcode/variant_annotator.py

```diff
@@ -11,7 +12,7 @@ class VariantAnnotator(object):
     def __init__(self, ensembl_release):
         self.ensembl = pyensembl.EnsemblRelease(ensembl_release)
 
-    def describe_variant(self, variant):
+    def describe_variant(self, variant, raise_on_error=True):
```

Comment:
The most common use case is `VariantCollection` which uses `VariantAnnotator` under the hood. Not sure how to economize on `raise_on_error` since you'll need one arg in `VariantCollection` and another in `VariantAnnotator`.